bitbadges/assetlist.json

```diff
@@ -19,11 +20,13 @@
       "display": "badge",
       "symbol": "BADGE",
       "logo_URIs": {
-        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitbadges/images/badge_logo.png"
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitbadges/images/badge_logo.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitbadges/images/badge_logo.svg"
```

Comment:
In the new image, please remove the black background and ensure the canvas goes right to the edge of the icon (remove padding).
Also, the uploaded image name has a typo: "baadge_logo.svg"